scrapy/spiders/sitemap.py

```diff
@@ -49,6 +49,13 @@ def _parse_sitemap(self, response):
                 logger.warning("Ignoring invalid sitemap: %(response)s",
                                {'response': response}, extra={'spider': self})
                 return
+            if response.url.endswith('.txt'):
+                for loc in iterloc_text_sitemap(response.text):
```

Comment:
:lipstick:  This does not seem worth a new function to me:

```suggestion
                for loc in response.text.splitlines():
```